code/API_definitions/device-reachability-status.yaml

```diff
@@ -0,0 +1,397 @@
+openapi: 3.0.3
+info:
+  title: Device Reachability Status Retrieval
+  description: |
+    This API provides the customer with the ability to query device:
+      - Reachability status
+
+    # Introduction
+
+    ## Reachability Status
+
+    API consumer is able to verify whether a certain user device is reachable from the network via data- or sms-usage.
+
+    # Relevant terms and definitions
+
+    * **Device**: A device refers to any physical entity that can connect to a network and participate in network communication.
+      At least one identifier for the device (user equipment) out of four options: IPv4 address, IPv6 address, Phone number, or Network Access Identifier assigned by the mobile network operator for the device.
+
+    * **Reachability** : Reachability status.
+      - `CONNECTED_SMS`, if device is connected to the network via SMS usage
+      - `CONNECTED_DATA`, if device is connected to the network via data usage
+      - `NOT_CONNECTED`, if device is not connected to the network
+
+    * **LastStatusTime** : This property specifies the time when the status was last updated. Its presence in the response indicates the freshness of the information, while its absence implies the information may be outdated or its freshness is uncertain.
+
+    # API Functionality
+
+    The API exposes following capabilities:
+
+    ## Device Reachability situation
+
+    The endpoint `POST /retrieve-reachability-status` allows to get current connectivity status information synchronously.
+
+
+    ## Further info and support
+
+    (FAQs will be added in a later version of the documentation)
+
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: wip
+externalDocs:
+  description: Product documentation at CAMARA
+  url: https://github.com/camaraproject/
+
+servers:
+  - url: "{apiRoot}/reachability-retrieval/v0"
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+tags:
+  - name: Device reachability status
+    description: Operations to get the current reachability status of a device
+paths:
+  /retrieve-reachability-status:

```

Comment:
@bigludo7 I see that for example in location-retrieval the WG opted for "/retrieval", in location-verification for "/verify", etc
If we have reachability-retrieval maybe we can go with "/retrieval" too? wdyt

Reply:
Hello @fernandopradocabrillo 
Yes make sense !
For the API name, today is `reachability-retrieval` - do you think we need to shift to `reachability-status` and then have an endpoint `/retrieve` - This is perhaps more straighforward?

Same will happen for roaming.

Reply:
sounds good to me, it looks clearer

Reply:
/retrieve would be good. 
API name: I would prefer device-reachability-status, which contains the context 'device' for the case other reachability status API will come later.